config/global.php

```diff
@@ -64,5 +64,7 @@
 
     'observers.global' => array(),
 
-    'Piwik\EventDispatcher' => DI\object()->constructorParameter('observers', DI\get('observers.global'))
+    'Piwik\EventDispatcher' => DI\object()->constructorParameter('observers', DI\get('observers.global')),
+
+    'Zend_Validate_EmailAddress' => DI\object('Zend_Validate_EmailAddress'),
```

Comment:
FYI this line is not necessary: by default with [autowiring](http://php-di.org/doc/autowiring.html) PHP-DI can create the instance without configuration needed (because here there is no constructor parameter to configure). It's no big deal but it's just to let you know